web_development_101/the_basics/introduction_to_web_development.md

```diff
@@ -53,43 +52,43 @@ These are some of the basic tools you will use regularly. You may not know what
 
 ### Motivation
 
-Learning to code is incredibly rewarding but it can also be difficult and frustrating.
+Learning to code is incredibly rewarding but can also be difficult and frustrating.
```

Comment:
```suggestion
Learning to code is incredibly rewarding, but can also be difficult and frustrating.
```

Reply:
@mindovermiles262 The comma that you've suggested to add in here is actually incorrect. It's an example of a compound predicate, where the subject ("Learning to code") takes two verbs ("is" and "can be"). In these situations, you don't want to put a comma between the subject and the verb. (For reference, see Chicago Manual of Style 6.29.) Alternatively, if you'd prefer to see a comma in this sentence, it could be changed to the following (adding in a second subject ("it") to make a compound sentence instead of a compound predicate):
"Learning to code is incredibly rewarding, but it can also be difficult and frustrating."